src/openvic-simulation/economy/ProductionType.cpp

```diff
@@ -29,96 +32,99 @@ bool ProductionType::parse_scripts(GameManager const& game_manager) {
 
 ProductionTypeManager::ProductionTypeManager() : rgo_owner_sprite { 0 } {}
 
-node_callback_t ProductionTypeManager::_expect_employed_pop(
-	GoodManager const& good_manager, PopManager const& pop_manager, callback_t<EmployedPop&&> cb
+node_callback_t ProductionTypeManager::_expect_job(
+	GoodManager const& good_manager, PopManager const& pop_manager, callback_t<Job&&> cb
 ) {
 
 	return [this, &good_manager, &pop_manager, cb](ast::NodeCPtr node) -> bool {
 		std::string_view pop_type;
-		EmployedPop::effect_t effect;
-		fixed_point_t effect_multiplier = 1, amount = 1;
+		Job::effect_t effect_type;
```

Comment:
```suggestion
		Job::effect_t effect_type = THROUGHPUT;
```
Just to be on the safe side, this should have a default value, so it doesn't contain nonesense data if `effect` is missing or invalid. If you move `using enum Job::effect_t;` from a couple lines below to the top of the file you won't need to write the enum's `Job::effect_t::` prefix.